public/assets/scripts/fetch-gallery-data.mjs

```diff
@@ -86,20 +93,39 @@ export function loadDomainContent(event) {
     });
 }
 
+
 function loadDomainPage(doc) {
+    const headBanner = document.querySelector("head-banner");
+
+    // Hide the main title, tagline, and button within the head-banner
+    const titleElement = headBanner.querySelector(".page-name");
+    const tagLineElement = headBanner.querySelector("h2");
+    const buttonElement = headBanner.querySelector("button");
+
+    if (titleElement) titleElement.style.display = "none";
+    if (tagLineElement) tagLineElement.style.display = "none";
+    if (buttonElement) buttonElement.style.display = "none";
```

Comment:
The exact same code exists in `loadFunctionDoc`, so best to put this into a helper function.